rackcity/views/asset_views.py

```diff
@@ -130,7 +130,7 @@ def asset_detail(request, id):
             return response
     try:
         if change_plan:
-            assets, assets_cp = get_assets_for_cp(change_plan.id)
+            assets, assets_cp = get_assets_for_cp(change_plan.id, show_decommissioned=True)
```

Comment:
is the decommission view set on the front end using the is_decommissioned flag?

Reply:
yeah -> if an asset has been decommissioned you still want to see the detailview page. I added a warning that shows that it has been decomissioned and disabled edit/delete buttons